lane.py

```diff
@@ -1157,7 +1158,6 @@ def search(self, query, search_client, pagination=None):
                         lazyload(mw.license_pool, LicensePool.identifier),
                         lazyload(mw.license_pool, LicensePool.presentation_edition),
                     )
-                    q = self.only_show_ready_deliverable_works(q, mw)
```

Comment:
This branch only does the paginated lists for search results, but I agree about the featured lanes. We can add that to the search filter if we start using it for that.